### ISSUE TITLE
Back arrow doesn't look clickable

### ISSUE THREAD

**Issue by aronasorman** [learningequality/ka-lite#3223]
Browser: Firefox 35
Branch: develop

Compare the cursor between the two pics:

Cursor on topic --
![screen shot 2015-03-10 at 1 18 21 pm](https://cloud.githubusercontent.com/assets/191955/6584381/12cec718-c728-11e4-9211-864bac7efd33.png)

Cursor on back button --
![screen shot 2015-03-10 at 1 18 34 pm](https://cloud.githubusercontent.com/assets/191955/6584393/29b31e20-c728-11e4-8ad5-8fe7ec22ca3a.png)

Basically, people using the mouse would have a hard time knowing that the back button region is clickable.

**Comment by Antrikshy** [comment]
Ooh. I should add a hover color change in that smaller CSS PR I have open.

**Comment by 66eli77** [comment]
@Antrikshy  this feature is specifically useful on mobile, please avoid hover as you can

**Comment by Antrikshy** [comment]
@66eli77 But hover won't matter on mobile.

The tapping functionality works in exactly the same way. Pretty much every beautiful mobile-friendly responsive website has tons of hover everywhere. While hover does not work on mobile platforms, it does not impede tapping functionality and I don't think it's frowned upon. I can't think of another way to make it look clickable for desktop users let alone mobile users anyway. Suggestions?
